src/services/completions.ts

```diff
@@ -1877,6 +1875,9 @@ namespace ts.Completions {
                 case SyntaxKind.YieldKeyword:
                 case SyntaxKind.TypeKeyword:  // type htm|
                     return true;
+
+                case SyntaxKind.AsteriskToken:

```

Comment:
Funny enough, it shouldn't be in a method

```ts
class C {
  *foo() {
      yield 10;
  }
}

class D extends C {
  */*1*/
}

let x: C = {
    */*2*/
}
```

Even though we don't provide anything, these should be new definition locations.